AgentQnA/README_miscellaneous.md

```diff
@@ -0,0 +1,50 @@
+# Agents for Question and Answering Application
+
+## Table of contents
+
+1. [Build Service Docker Image](#build-service-docker-image)
+2. [Build UI Docker Image](#build-ui-docker-image)
+3. [Generate a HuggingFace Access Token](#generate-a-huggingface-access-token)
+4. [Monitor and Tracing](#monitor-and-tracing)
```

Comment:
If `README.md` is supposed to be end-user documentation, and `README_miscellaneous.md` develoment information, I think it would be better to rename latter to something like `BUILD.md` or `Development.md`.

Although access token section talks about what developer does, access token is needed also to **use** the service, not just to develop it.  And monitoring is also a production use-case, not just development one. I.e. to me both seems ot have relevance for end-user doc.